src/main.ts

```diff
@@ -508,17 +512,25 @@ export default class WhatsApp {
 
     this.keyPair = null;
 
-    this.qrPath = resolvePath(".", qrPath);
+    this.parameters = {...{
+      keysPath : "./keys.json",
+      qrPath : "./qrcode.png",
+      clientName : "WhatsApp forwarder",
+      clientShortName : "WhatsAppForwarder",
+      restoreSession : false
+    }, ...parameters}
+
+    this.parameters.qrPath = resolvePath(".", this.parameters.qrPath);
```

Comment:
```suggestion
    this.opts.qrPath = resolvePath(".", this.opts.qrPath);
```
Update the property name